### ISSUE TITLE
Preferred Countries list

### ISSUE THREAD

**Issue by azeemkhalid04** [nikhiln/ngx-international-phone-number#41]
It would be great if you can provide us an option to show some of counties on top of the list in drop down. There may be an option to pass array of preferredCountries[] which will be listed on top. Please see attachment.

Thanks in advance.

![image](https://user-images.githubusercontent.com/36879875/57680378-e7c8de00-7646-11e9-8e8a-321e1c29f6e2.png)

**Comment by Michaelwan777** [comment]
This feature is something like in [intl-tel-input](https://github.com/jackocnr/intl-tel-input)

**Comment by azeemkhalid04** [comment]
> This feature is like something in [intl-tel-input](https://github.com/jackocnr/intl-tel-input).

Exactly, but intl-tel-input is not angular compatible.

**Comment by Michaelwan777** [comment]
@azeemkhalid04 Yep, I used to use intl-tel-input. However, it has some bugs in validating phone number... I'm working on how to make a preferred coutries list based on this plug-in.

**Comment by choopage** [comment]
@esoyke might fix this for you. https://github.com/esoyke/ngx-international-phone-number2
Put this there. He might help.

**Comment by Michaelwan777** [comment]
> > This feature is like something in [intl-tel-input](https://github.com/jackocnr/intl-tel-input).
> 
> Exactly, but intl-tel-input is not angular compatible.
This may help.
https://github.com/jackocnr/intl-tel-input/issues/802#issuecomment-564814518
